package.json

```diff
@@ -19,7 +19,7 @@
     "build": "node ./scripts/build.js",
     "build-storybook": "build-storybook",
     "prepublishOnly": "pinst --disable && node ./scripts/prepublish.js",
-    "postinstall": "husky install && cd node_modules/web-components && yarn install --production && npm run-script build",
+    "postinstall": "husky install",
```

Comment:
With the `prepare` script that was part  of the most recent release, we no longer need to install + build in the `postinstall`